Docker containers won't build locally

Just pulled the latest changes and am seeing this:

```
 => ERROR [dev 22/24] RUN yarn install --frozen-lockfile                                                                                                                           22.5s
------                                                                                                                                                                                   
 > [dev 22/24] RUN yarn install --frozen-lockfile:                                                                                                                                       
#31 0.285 yarn install v1.22.19                                                                                                                                                          
#31 0.355 [1/4] Resolving packages...                                                                                                                                                    
#31 0.560 [2/4] Fetching packages...                                                                                                                                                     
#31 16.06 [3/4] Linking dependencies...                                                                                                                                                  
#31 16.06 warning " > react-select@4.3.0" has unmet peer dependency "react@^16.8.0 || ^17.0.0".
#31 16.06 warning " > react-select@4.3.0" has unmet peer dependency "react-dom@^16.8.0 || ^17.0.0".
#31 16.06 warning "react-select > @emotion/react@11.1.5" has unmet peer dependency "react@>=16.8.0".
#31 16.06 warning "react-select > react-input-autosize@3.0.0" has unmet peer dependency "react@^16.3.0 || ^17.0.0".
#31 16.06 warning "react-select > react-transition-group@4.4.1" has unmet peer dependency "react@>=16.6.0".
#31 16.06 warning "react-select > react-transition-group@4.4.1" has unmet peer dependency "react-dom@>=16.6.0".
#31 16.07 warning "workspace-aggregator-4afcadde-a121-4578-805d-2a2dc99a2ecc > @experimenter/core > eslint-plugin-jest > @typescript-eslint/experimental-utils > @typescript-eslint/typescript-estree > tsutils@3.21.0" has unmet peer dependency "typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta".
#31 20.71 [4/4] Building fresh packages...
#31 21.82 info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
#31 21.82 error /app/node_modules/deasync: Command failed.
#31 21.82 Exit code: 1
#31 21.82 Command: node ./build.js
#31 21.82 Arguments: 
#31 21.82 Directory: /app/node_modules/deasync
#31 21.82 Output:
#31 21.82 gyp info it worked if it ends with ok
#31 21.82 gyp info using node-gyp@5.1.0
#31 21.82 gyp info using node@14.17.1 | linux | arm64
#31 21.82 gyp info find Python using Python version 3.11.1 found at "/usr/local/bin/python"
#31 21.82 gyp http GET https://nodejs.org/download/release/v14.17.1/node-v14.17.1-headers.tar.gz
#31 21.82 gyp http 200 https://nodejs.org/download/release/v14.17.1/node-v14.17.1-headers.tar.gz
#31 21.82 gyp http GET https://nodejs.org/download/release/v14.17.1/SHASUMS256.txt
#31 21.82 gyp http 200 https://nodejs.org/download/release/v14.17.1/SHASUMS256.txt
#31 21.82 gyp info spawn /usr/local/bin/python
#31 21.82 gyp info spawn args [
#31 21.82 gyp info spawn args   '/root/.nvm/versions/node/v14.17.1/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
#31 21.82 gyp info spawn args   'binding.gyp',
#31 21.82 gyp info spawn args   '-f',
#31 21.82 gyp info spawn args   'make',
#31 21.82 gyp info spawn args   '-I',
#31 21.82 gyp info spawn args   '/app/node_modules/deasync/build/config.gypi',
#31 21.82 gyp info spawn args   '-I',
#31 21.82 gyp info spawn args   '/root/.nvm/versions/node/v14.17.1/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
#31 21.82 gyp info spawn args   '-I',
#31 21.82 gyp info spawn args   '/root/.cache/node-gyp/14.17.1/include/node/common.gypi',
#31 21.82 gyp info spawn args   '-Dlibrary=shared_library',
#31 21.82 gyp info spawn args   '-Dvisibility=default',
#31 21.82 gyp info spawn args   '-Dnode_root_dir=/root/.cache/node-gyp/14.17.1',
#31 21.82 gyp info spawn args   '-Dnode_gyp_dir=/root/.nvm/versions/node/v14.17.1/lib/node_modules/npm/node_modules/node-gyp',
#31 21.82 gyp info spawn args   '-Dnode_lib_file=/root/.cache/node-gyp/14.17.1/<(target_arch)/node.lib',
#31 21.82 gyp info spawn args   '-Dmodule_root_dir=/app/node_modules/deasync',
#31 21.82 gyp info spawn args   '-Dnode_engine=v8',
#31 21.82 gyp info spawn args   '--depth=.',
#31 21.82 gyp info spawn args   '--no-parallel',
#31 21.82 gyp info spawn args   '--generator-output',
#31 21.82 gyp info spawn args   'build',
#31 21.82 gyp info spawn args   '-Goutput_dir=.'
#31 21.82 gyp info spawn args ]
#31 21.82 Traceback (most recent call last):
#31 21.82   File "/root/.nvm/versions/node/v14.17.1/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py", line 50, in <module>
#31 21.82     sys.exit(gyp.script_main())
#31 21.82              ^^^^^^^^^^^^^^^^^
#31 21.82   File "/root/.nvm/versions/node/v14.17.1/lib/node_modules/npm/node_modules/node-gyp/gyp/pylib/gyp/__init__.py", line 554, in script_main
#31 21.82     return main(sys.argv[1:])
#31 21.82            ^^^^^^^^^^^^^^^^^^
#31 21.82   File "/root/.nvm/versions/node/v14.17.1/lib/node_modules/npm/node_modules/node-gyp/gyp/pylib/gyp/__init__.py", line 547, in main
#31 21.82     return gyp_main(args)
#31 21.82            ^^^^^^^^^^^^^^
#31 21.82   File "/root/.nvm/versions/node/v14.17.1/lib/node_modules/npm/node_modules/node-gyp/gyp/pylib/gyp/__init__.py", line 520, in gyp_main
#31 21.82     [generator, flat_list, targets, data] = Load(
#31 21.82                                             ^^^^^
#31 21.82   File "/root/.nvm/versions/node/v14.17.1/lib/node_modules/npm/node_modules/node-gyp/gyp/pylib/gyp/__init__.py", line 136, in Load
#31 21.82     result = gyp.input.Load(build_files, default_variables, includes[:],
#31 21.82              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
#31 21.82   File "/root/.nvm/versions/node/v14.17.1/lib/node_modules/npm/node_modules/node-gyp/gyp/pylib/gyp/input.py", line 2782, in Load
#31 21.82     LoadTargetBuildFile(build_file, data, aux_data,
#31 21.82   File "/root/.nvm/versions/node/v14.17.1/lib/node_modules/npm/node_modules/node-gyp/gyp/pylib/gyp/input.py", line 391, in LoadTargetBuildFile
#31 21.82     build_file_data = LoadOneBuildFile(build_file_path, data, aux_data,
#31 21.82                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
#31 21.82   File "/root/.nvm/versions/node/v14.17.1/lib/node_modules/npm/node_modules/node-gyp/gyp/pylib/gyp/input.py", line 234, in LoadOneBuildFile
#31 21.82     build_file_contents = open(build_file_path, 'rU').read()
#31 21.82                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
#31 21.82 ValueError: invalid mode: 'rU' while trying to load binding.gyp
#31 21.82 gyp ERR! configure error 
#31 21.82 gyp ERR! stack Error: `gyp` failed with exit code: 1
#31 21.82 gyp ERR! stack     at ChildProcess.onCpExit (/root/.nvm/versions/node/v14.17.1/lib/node_modules/npm/node_modules/node-gyp/lib/configure.js:351:16)
#31 21.82 gyp ERR! stack     at ChildProcess.emit (events.js:375:28)
#31 21.82 gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:277:12)
#31 21.82 gyp ERR! System Linux 5.15.49-linuxkit
#31 21.82 gyp ERR! command "/root/.nvm/versions/node/v14.17.1/bin/node" "/root/.nvm/versions/node/v14.17.1/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
#31 21.82 gyp ERR! cwd /app/node_modules/deasync
#31 21.82 gyp ERR! node -v v14.17.1
#31 21.82 gyp ERR! node-gyp -v v5.1.0
#31 21.82 gyp ERR! not ok 
#31 21.82 Build failed
------
executor failed running [/bin/sh -c yarn install --frozen-lockfile]: exit code: 1
make: *** [build_test] Error 1
```

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-3000)

--- a/app/Dockerfile
+++ b/app/Dockerfile
@@ -26,7 +26,7 @@ RUN chmod +x /app/bin/wait-for-it.sh
 
 
 # Install nvm with node and npm
-ENV NODE_VERSION=14.17.1
+ENV NODE_VERSION=16.19.0
 RUN curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.35.3/install.sh | bash
 ENV NVM_DIR=/root/.nvm
 RUN . "$NVM_DIR/nvm.sh" && nvm install ${NODE_VERSION}

--- a/app/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx
+++ b/app/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx
@@ -1142,7 +1142,7 @@ describe("FormAudience", () => {
     });
 
     expect(
-      within(screen.queryByTestId("audience-values") as HTMLElement)
+      within(screen.queryByTestId("population-percent-top-row") as HTMLElement)
         .findByText(expectedValue, {
           selector: `[data-for={populationPercent}]`,
         })

